src/app/clients/[id]/ViewClientPage.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import React from "react";
+import LinkButton from "@/components/Buttons/LinkButton";
+import styled from "styled-components";
+import { CenterComponent, StyledCard } from "@/components/Form/formStyling";
+import { ClientData } from "@/app/clients/[id]/getClientData";
+import { formatCamelCaseKey } from "@/pdf/ShoppingList/dataPreparation";
+
+interface Props {
+    clientID: string;
+    clientData: ClientData;
+}
+
+const NormalText = styled.pre`
+    font-family: inherit;
+    font-weight: normal;
+    padding-bottom: 1rem;
+    width: inherit;
+    word-break: break-word;
+`;
+
+const TempDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    place-items: baseline;
+    width: inherit;
+`;
+
+const KeyText = styled.p`
+    font-weight: bold;
+`;
+
+const ViewClientPage: React.FC<Props> = ({ clientID, clientData }) => {
+    return (
+        <CenterComponent>
+            <StyledCard>
+                <TempDiv>
```

Comment:
does both inner and outer need to be wrapped in TempDivs?

Reply:
thanks for catching this TT 
neither of them are needed - hence the `Temp` in the name ^^